Streetcode/Streetcode.BLL/DTO/Authentication/Register/RegisterRequestDTO.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using Streetcode.BLL.Attributes.Authentication;
+using Streetcode.BLL.Constants.Authentication;
+
+namespace Streetcode.BLL.DTO.Authentication.Register;
+
+public class RegisterRequestDTO
+{
+    [Required]
+    [MaxLength(50)]
+    [DefaultValue(AuthConstants.NAME)]
+    public string Name { get; set; }
+
+    [Required]
+    [MaxLength(50)]
+    [DefaultValue(AuthConstants.SURNAME)]
+    public string Surname { get; set; }
+
+    [Required]
+    [EmailAddress]
+    [ValidEmail]
+    [DefaultValue(AuthConstants.EMAIL)]
+    public string Email { get; set; }
+
+    [Required]
+    [MaxLength(50, ErrorMessage = "UserName maximum length is 50")]
+    [DefaultValue(AuthConstants.USERNAME)]
+    public string UserName { get; set; }
+
+    [Required]
+    [Display(Name = "Phone Number")]
+    [Phone]
+    [DefaultValue(AuthConstants.PHONENUMBER)]
+    public string PhoneNumber { get; set; }

```

Comment:
Do we really need the phone number to be required? As far as I know it can be null in AspNetUsers table, can't it?